pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -137,10 +163,33 @@ def work(self):
                     'chain_hack_sequence_number']
             else:
                 logger.log('Possibly searching too often - taking a short rest :)', 'yellow')
+                self.bot.fort_timeouts[self.fort["id"]] = (time.time() + 300) * 1000  # Don't spin for 5m
                 return 11
         sleep(8)
         return 0
 
+    def catch_pokemon(self, pokemon):
+        worker = PokemonCatchWorker(pokemon, self)
+        return_value = worker.work()
+
+        if return_value == PokemonCatchWorker.BAG_FULL:
+            worker = InitialTransferWorker(self)
+            worker.work()
+
+        return return_value
+
+    def drop_items(self):
+        self.api.get_player()
+        response_dict = self.api.call()
+        player = response_dict['responses']['GET_PLAYER']['player_data']
+        items_count = self.bot.get_inventory_count('item')
+        items_stock = self.current_inventory()
+        worker = DropItemsWorker(self.bot, items_count, player['max_item_storage'], items_stock.copy())
```

Comment:
It shouldn't be called from anywhere except work_on_cell in __init__.py. Next to the call to the InitialTransferWorker